Adamant/Modules/Chat/View/ChatViewController.swift

```diff
@@ -943,6 +943,7 @@ private extension ChatViewController {
     func closeFileToolbarView() {
         filesToolbarView.removeFromSuperview()
         messageInputBar.invalidateIntrinsicContentSize()
+        viewModel.inputTextFileUpdated()
```

Comment:
Let's not do it here. 

`@ObservableValue var filesPicked: [FileResult]?`

We can easily observe new files from the ChatViewModel